src/java.desktop/share/classes/sun/awt/image/URLImageSource.java

```diff
@@ -40,21 +39,7 @@ public class URLImageSource extends InputStreamImageSource {
     int actualPort;
 
     public URLImageSource(URL u) {
-        @SuppressWarnings("removal")
-        SecurityManager sm = System.getSecurityManager();
-        if (sm != null) {
-            try {
-                java.security.Permission perm =
-                      URLUtil.getConnectPermission(u);
```

Comment:
`URLUtil.getConnectPermission(u);` is no longer used anywhere and can be removed


https://github.com/openjdk/jdk/commit/8c644ca310c44995a1fc899eabb5611b79873c68#diff-50e0953b57d7ba2dbb8faa9b1766b99234d5ae4cb271c48ece13b9e3d0c9ca89

so the jdk/src/java.base/share/classes/sun/net/util/URLUtil.java part can be reverted

Reply:
Will changes to java.base files - URLUtil.java be handled in [JDK-8344180](https://bugs.openjdk.org/browse/JDK-8344180)? Or do we do it here?

Reply:
> Will changes sun/net/util/URLUtil.java be handled in [JDK-8344180](https://bugs.openjdk.org/browse/JDK-8344180)? 

It has only two subtasks with mentions of `URLUtil`:

- [JDK-8344221](https://bugs.openjdk.org/browse/JDK-8344221) it is already resolved
- this issue

So I guess this removal can be the part of this PR

Reply:
@prrace There is no other code using it, the 2 awt.sun files were the only two files that were using URLUtil.getConnectPermission().

@dfuch I can delete the unused methods in sun/net/util/URLUtil.java and add [JDK-8344180](https://bugs.openjdk.org/browse/JDK-8344180) as additional issue to this PR and handle it all-together. Please let me know if the above works or if you want to handle URLUtil.java changes separately.

Reply:
Hi @azvegint, if you have no objection to include the deletion of `URLUtil.getConnectPermission()` in this PR, then please do.